README.md

```diff
@@ -23,11 +23,24 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
```

Comment:
We need to mention here that the integration backwards-compatibility is provided by Cloudflare, and there are only requirements for local development. We can mention [this field](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/rc/wrangler.toml#L6) as well.

Reply:
Updated with even better, as Juraj Suggested I extended requirements section with two subsections called "Usage Requirements" and "Development Requirements".